script/rbenv-install.sh

```diff
@@ -2,13 +2,24 @@
 #
 # Install our selected Ruby version defined in the .ruby-version file.
 #
-# Requires:
+# Requires and tries to install if missing:
 # - [rbenv](https://github.com/rbenv/rbenv#readme)
 # - [ruby-build](https://github.com/rbenv/ruby-build#readme)
 #
 # If our ruby-build version is outdated and it can't build the version we want
 # then we try upgrading ruby-build and installing again.
 
+if ! command -v rbenv > /dev/null; then
+  # Install rbenv:
+  git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+  ~/.rbenv/bin/rbenv init
+  eval "$(rbenv init -)"
```

Comment:
Hmm the current shell hasn't loaded the updated bash profile yet, so we probably need the full path here too:
```suggestion
  eval "$(~/.rbenv/bin/rbenv init -)"
```